modules/draco/src/draco-loader.js

```diff
@@ -40,7 +40,8 @@ async function parse(arrayBuffer, options, context, loader) {
     // TODO passing in options causes CI failures...
     // @ts-ignore
     return dracoParser.parseSync(arrayBuffer, {
-      extraAttributes: (options.draco && options.draco.extraAttributes) || null
+      extraAttributes: (options.draco && options.draco.extraAttributes) || null,
```

Comment:
Hmm, this should be handled by default arguments for `options.draco.extraAttributes`. I merged one fix already today.

Reply:
This PR is from your branch "ib/draco-metadata" into "ib/draco-metadata. What fix do you mean?